tools/pipelines/build-docs.yml

```
@@ -121,7 +156,7 @@ stages:
             echo latestPipeline ${{ variables.latestPipeline }}
             echo n1Pipeline ${{ variables.n1Pipeline }}
             echo repoToTrigger ${{ variables.repoToTrigger }}
-            echo shouldDeploy ${{ variables.shouldDeploy }}
+            echo shouldDeploy $(dependencies.check_branch_version.outputs['check_branch_version.SetShouldDeploy.shouldDeploy'])
```

nit:

I’d suggest using `stageDependencies` instead of `dependencies`. While dependencies work in this case since the stages are directly connected via dependOn, I find it clearer to use `stageDependencies` for stage-to-stage outputs and `dependencies` for job level. :)

```suggestion
            echo shouldDeploy $(stageDependencies.check_branch_version.outputs['check_branch_version.SetShouldDeploy.shouldDeploy'])
```